wicketd/src/update_tracker.rs

```diff
@@ -2518,6 +3000,7 @@ impl<'a> SpComponentUpdateContext<'a> {
             }
             UpdateComponent::Sp => {
                 // Nothing special to do on the SP - just reset it.
+                // TODO fixup the SP to also set the active slot
```

Comment:
Is this for oxidecomputer/hubris#1808? Do we need to put this fix in before R9 and/or were you planning to do it on this PR?

Reply:
Yes, this is related. It's not necessary for this PR to work and I was wary about adding more logic changes for backwards compatability since this PR is already pretty large.